src/main/steps/respondent/international-factors/start/content.ts

```diff
@@ -51,7 +51,7 @@ export const form: FormContent = {
       classes: 'govuk-radios',
       label: l => l.label,
       section: l => l.section,
-      hint: l => l.hint,
+      hint: l => l.twoHint,
```

Comment:
Can't we just keep it as 'hint'? 'twoHint' doesn't make sense here.

Reply:
code updated with label as hint instead of twoHint